Framework/Algorithms/src/MaxEnt.cpp

```diff
@@ -204,6 +205,30 @@ void MaxEnt::init() {
                                                          500, mustBePositive,
                                                          Direction::Input),
                   "Maximum number of iterations in alpha chop.");
+  declareProperty(
+      make_unique<WorkspaceProperty<>>(
+          "DataLinearAdj", "", Direction::Input, PropertyMode::Optional,
+          boost::make_shared<EqualBinSizesValidator>(errorLevel, warningLevel)),
+      "Adjusts the calculated data by multiplying each value by the "
+      "corresponding Y value of this workspace. "
+      "The data in this workspace is complex in the same manner as complex "
+      "input data.");
+  declareProperty(
+      make_unique<WorkspaceProperty<>>(
+          "DataConstAdj", "", Direction::Input, PropertyMode::Optional,
+          boost::make_shared<EqualBinSizesValidator>(errorLevel, warningLevel)),
+      "Adjusts the calculated data by adding to each value the corresponding Y "
+      "value of this workspace. "
+      "If DataLinearAdj is also specified, this addition is done after its "
+      "multiplication. "
```

Comment:
Why have you added the sentence 'If DataLinearAdj is also specified, this addition is done after its "
+      "multiplication' ?

Reply:
I needed to state the order of the two operations multiplication and addition, because they do not commute.

Reply:
I see what you mean now. However, you have in the documentation a nice equation showing how these properties are applied. Hence, I would argue that either remove this sentence or add 'See equation in documentation for how DataLinearAdj and DataConstAdj are applied'. What do you think

Reply:
I think it should stay in. It saves the reader from having to go to the equation in the documentation.

Reply:
yes in theory:-) but only a few of our users will have the mathematically background to instantly understand what it means

Reply:
but they will understand the equation.

Reply:
I've added the "See equation ...".